.gitignore

```diff
@@ -12,7 +12,6 @@
 /spec/examples.txt
 /test/tmp/
 /test/version_tmp/
-/tmp/
```

Comment:
Do we need this change?

Reply:
Nope. I think this was leftover. It important to ignore any tmp files we might generate but shouldn't be the case since we `.unlink` or delete any tmp files generated.